tests/test_insert.py

```diff
@@ -195,3 +198,44 @@ def test_single_insert_obs_not_unique(test_session):
     ]
     dbm = single_insert_strategy(test_session, ob)
     assert dbm.skips == 1
+
+
+def test_mult_networks(test_session, caplog):
+    caplog.set_level(logging.DEBUG, "crmprtd")
+
+    moti = "Brandywine"
+    wmb = "FIVE MILE"
+    ec = "Sechelt"
+    stations = [moti, wmb, ec]
+
+    obs = []
+    with test_session:
+        for station in stations:
+            hist = (
+                test_session.query(History)
+                .filter(History.station_name == station)
+                .first()
+            )
+            var = hist.station.network.variables[0]
+            observation = Obs(
+                history_id=hist.id,
+                datum=10,
+                vars_id=var.id,
+                time=datetime(2012, 9, 24, 6, tzinfo=pytz.utc),
+            )
+            obs.append(observation)
+
+    results = insert(test_session, obs)
+    assert networks_logged(obs, test_session, caplog)
+
+
+def networks_logged(obs, test_session, caplog):
```

Comment:
If you rename this to be `networks_are_logged` it might read more clearly as to what it does.

And what about simply passing in a tuple of network names that you expect to be logged. It would simplify the signature:

```
def networks_are_logged(networks, caplog):
```